nautobot/core/settings.json

```diff
@@ -0,0 +1,598 @@
+{
+    "type": "object",
+    "definitions": {
+        "callable": {
+            "type": "string"
+        },
+        "regex": {
+            "type": "string"
+        }
+    },
+    "properties": {
+        "ADMINS": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "default": []
+        },
+        "ALLOWED_HOSTS": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "default": []
+        },
+        "BANNER_BOTTOM": {
+            "type": "string",
+            "default": ""
+        },
+        "BANNER_LOGIN": {
+            "type": "string",
+            "default": ""
+        },
+        "BANNER_TOP": {
+            "type": "string",
+            "default": ""
+        },
+        "BRANDING_FILEPATHS": {
+            "type": "object",
+            "properties": {
+                "logo": {
+                    "type": "string",
```

Comment:
Wonder if it'd be useful to define a `relative_path` type to use for these properties and similar ones?